Gemfile

```diff
@@ -26,7 +26,7 @@ end
 
 group :development do
   # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
-  gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'listen'
```

Comment:
consider doing `~> 3.7` as the version requirement here

Reply:
done.  I thought we were avoiding pinning versions unless necessary, and if necessary we were leaving a eason.  Saves some churn, no?

Reply:
I don't think that '~> 3.7' is a pin as it allows any version up to 4.0.